src/Core/Animation/Animation.hpp

```diff
@@ -1,22 +1,21 @@
 #ifndef ANIMATION_HPP
 #define ANIMATION_HPP
 
-#include <Core/Animation/Pose/Pose.hpp>
+#include <Core/Animation/Pose.hpp>
 #include <vector>
 
 namespace Ra {
 namespace Core {
 namespace Animation {
 
-using KeyPose = std::pair<Scalar, Pose>;
-
 class RA_CORE_API Animation {
```

Comment:
should it be merged with `Ra::Core::Animation::KeyPose` ?

Reply:
As they were separated, I didn't want to merge them to avoid side effects.
It is true this needs to be fixed, but can be done later (should be).